templates/terraform/examples/cloudbuild_trigger_github.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  trigger_template {
+    branch_name = "master"
+    repo_name   = "my-repo"
```

Comment:
I think a trigger can either have `trigger_template` or `github` object but not both. Please double check this.

Reply:
In general, the docs will say which fields are exclusive, and they don't on this one, but I'll take a look.

Reply:
Correct! Only one of these can be set at a time. I'll make sure the API docs get updated to reflect this.